config/default_records/plate_purposes/005_limber_purposes.yml

```diff
@@ -77,3 +77,7 @@ LTN Lib PCR XP:
 RVI Lib PCR XP:
   type: PlatePurpose
   stock_plate: false
+
+# A new plate purpose for multiplexed sequencing submissions,
+# avoiding FC on library quant submission.
+Library Stock Plate: *limber_input_96
```

Comment:
what does FC stand for?

Reply:
Fake Cherrypicking - It is used on the Confluence page, so I thought it was a common abbreviation—maybe not after all

Reply:
ah, fake cherrypick. I think a better description might be something like:

```
# A plate provided as ready-made libraries.
# These used to be imported as Stock Plates and then 'fake cherrypicked' onto 'LB-Lib-PCR-XP', which were allowed down the sequencing pipeline.
# Now, they can be imported and submitted for sequencing as this more accurate plate purpose.
```